go.mod

```diff
@@ -7,8 +7,8 @@ require (
 	github.com/ipfs/go-bitswap v0.1.8
 	github.com/ipfs/go-blockservice v0.1.2
 	github.com/ipfs/go-cid v0.0.3
-	github.com/ipfs/go-datastore v0.1.0
-	github.com/ipfs/go-ds-badger v0.0.5
+	github.com/ipfs/go-datastore v0.3.0
```

Comment:
I'm not totally familiar with the implications of this update and can revert it if it may introduce anything unexpected. it matches go-ds-badger's update here (https://github.com/ipfs/go-ds-badger/commit/133948da24fce6830c9d3a4ea73e66929ceda6a4)

Reply:
From 3.0 datastore must fulfil the `Sync` method

Reply:
should be ok to update to the latest version of these here.